dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/BitstreamMetadataValuePathUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.utils;
+
+import org.dspace.app.rest.exception.UnprocessableEntityException;
+import org.dspace.app.rest.submit.step.UploadStep;
+import org.dspace.app.util.DCInputSet;
+import org.dspace.app.util.DCInputsReader;
+import org.dspace.app.util.DCInputsReaderException;
+
+/**
+ * Utils class offering methods to validate patch operations for bitstream metadata in the submission
+ * 
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.it)
+ */
+public class BitstreamMetadataValuePathUtils {
+
+    private DCInputsReader inputReader;
+
+    BitstreamMetadataValuePathUtils() throws DCInputsReaderException {
+        inputReader = new DCInputsReader();
+    }
+
+    /**
+     * Method to verify that the path included in the patch operation is supported
+     * by the submission configuration of the upload section
+     * 
+     * @param absolutePath the path in the json patch operation
+     * @throws DCInputsReaderException      if an error occurs reading the
+     *                                      submission configuration
+     * @throws UnprocessableEntityException if the path is invalid
+     */
+    public void validate(String absolutePath) throws DCInputsReaderException {
+        String[] split = absolutePath.split("/");
+        DCInputSet inputConfig = inputReader.getInputsByFormName(UploadStep.UPLOAD_STEP_METADATA_SECTION);
+        if (split.length >= 4) {
```

Comment:
I'd appreciate an example or explanation in either the JavaDocs or in an inline comment to explain why we are looking for >=4 "parts" to the path.  It's not clear within this class what the path structure is expected to look like.  Also, there are no Unit Tests here to show the expected path structure, and which ones should validate and which should throw exceptions.